### ISSUE TITLE
IntelliSense should not occur when typing a period and Enter at the end of a comment

### ISSUE THREAD

**Issue by Lx** [loganch/AutoIt-VSCode#56]
## Steps to reproduce

1.  Open an AutoIt file.
2.  On a blank line, type this comment followed by <kbd>Enter</kbd>:

    ```autoit
    ; My first comment.
    ```

## Expected result

Cursor moves to next line and previous line is as shown:

```autoit
; My first comment.
```

## Actual result

Cursor remains on current line, the period is lost, and unexpected code appears both to the end of and underneath the comment:

```autoit
; a commentIf (<expression>) Then
	statement
EndIf
```

Affects version 1.0.4 with Code 1.43.0 (both latest at time of report), on both Windows and Mac.

**Comment by loganch** [comment]
Hey @Lx thanks for submitting the issue. I looked into it and it's due to the snippets I have in place for some of the code blocks. I took them almost straight from AutoIt documentation and see how including the ellipses can mess with IntelliSense. I'm going to convert them to something more descriptive so they'll show closer to the right context.

**Comment by loganch** [comment]
Just released v1.0.5 which addresses this.
